### ISSUE TITLE
Dockerize is timing out for the `gate` pod in `set-up-port-forwarding` command

### ISSUE THREAD

**Issue by austenLacy** [CircleCI-Public/spinnaker-orb#16]
### Orb version
0.3.0

### What happened
The `set-up-port-forwarding` command with the `wait` parameter set to true is not working. It seems that the `dockerize` command is timing out for the `gate` pod.

#### deck port forwarding output
```
Pod name found: spin-deck-7cb5c7d4c5-hp6zp
Forwarding from 127.0.0.1:9000 -> 9000
Handling connection for 9000
```

#### gate port forwarding output
```
Pod name found: spin-gate-6fcd474c7-r8228
Forwarding from 127.0.0.1:8084 -> 8084
```

#### dockerize command output
```
2020/01/03 16:33:08 Waiting for: http://localhost:9000
2020/01/03 16:33:08 Waiting for: http://localhost:8084
2020/01/03 16:33:08 Problem with request: Get http://localhost:8084: dial tcp 127.0.0.1:8084: connect: connection refused. Sleeping 1s
2020/01/03 16:33:08 Problem with request: Get http://localhost:9000: dial tcp 127.0.0.1:9000: connect: connection refused. Sleeping 1s
2020/01/03 16:33:09 Problem with request: Get http://localhost:9000: dial tcp 127.0.0.1:9000: connect: connection refused. Sleeping 1s
2020/01/03 16:33:09 Problem with request: Get http://localhost:8084: dial tcp 127.0.0.1:8084: connect: connection refused. Sleeping 1s
2020/01/03 16:33:11 Received 200 from http://localhost:9000
2020/01/03 16:33:11 Received 403 from http://localhost:8084. Sleeping 1s
2020/01/03 16:33:12 Received 403 from http://localhost:8084. Sleeping 1s
2020/01/03 16:33:13 Received 403 from http://localhost:8084. Sleeping 1s
```

It looks like that the `dockerize` command was able to talk to the `deck` pod on port `9000`, but timed out on the `gate` pod.

Running the port forward manually:
```
kubectl --namespace spinnaker port-forward spin-gate-6fcd474c7-r8228 8084:8084
Forwarding from 127.0.0.1:8084 -> 8084
Handling connection for 8084
```
and trying to `curl` the root throws the 403
```
curl http://localhost:8084
{"timestamp":1578069261540,"status":403,"error":"Forbidden","message":"Access Denied"}
```

### Expected behavior
It should try a valid URL other than the root that does not throw a 403.

An example could be the `/health` route. For example using the same port forwarding that threw the 403 in the previous test:

```
curl http://localhost:8084/health
{"status":"UP"}
```

**Comment by lokst** [comment]
@austenLacy Thanks so much for reporting this issue and for the great suggestion! We'll look into addressing this as soon as possible.

**Comment by austenLacy** [comment]
@lokst no problem! Thanks for the quick PR :)

**Comment by lokst** [comment]
@austenLacy The fix is now available in version 0.3.1 of the orb 🙂
